cmd/gateway/commands.go

```diff
@@ -320,6 +330,20 @@ func createControllerCommand() *cobra.Command {
 			` Lives in the same Namespace as the controller.`,
 	)
 
+	cmd.Flags().BoolVar(
+		&endpointPickerEnableTLS,
+		endpointPickerEnableTLSFlag,
+		true,
+		"Enables TLS when connecting to the endpoint picker.",
+	)
+
+	cmd.Flags().BoolVar(
+		&endpointPickerSkipSecureVerify,
+		endpointPickerSkipSecureVerifyFlag,
+		true,
+		"Disables server certificate verification when connecting to the endpoint picker, if TLS is enabled",
```

Comment:
Again let's leave an explanatory comment:
e.g.
`REQUIRED: Must be true until Gateway API Inference Extension EPP supports mounting certificates.`